src/main/scala/uk/gov/hmrc/apiplatform/modules/commands/applications/domain/models/ApplicationCommand.scala

```diff
@@ -67,6 +67,7 @@ object ApplicationCommands {
   case class UnsubscribeFromApi(actor: Actor, apiIdentifier: ApiIdentifier, timestamp: LocalDateTime)                                                   extends ApplicationCommand
   case class UpdateRedirectUris(actor: Actor, oldRedirectUris: List[String], newRedirectUris: List[String], timestamp: LocalDateTime)                   extends ApplicationCommand
   case class VerifyResponsibleIndividual(instigator: UserId, timestamp: LocalDateTime, requesterName: String, riName: String, riEmail: LaxEmailAddress) extends ApplicationCommand
+  case class IPAllowlistCIDRBlockChanged(actor: Actor, timestamp: LocalDateTime, oldIPAllowlist: List[String], newIPAllowlist: List[String])            extends ApplicationCommand
```

Comment:
Should the capitalisation be `Cidr` rather than `CIDR`?

Reply:
I really don't care - whatever people want is fine with me. I tend to uppercase acronyms so that folk know it's an acronym and not an actual word but i'm not fussed either way.

Reply:
I think I saw a style guide somewhere when I started ... was it my imagination?  I thought it's why we do `Mfa`, `Id`, `Uri`, etc.

Reply:
I wouldn't worry about it cos it'll change when the domain changes go in, this stuff will change

Reply:
:thinking: